CMakeLists.txt

```diff
@@ -70,12 +69,13 @@ if(MSVC)
     set(ROOT_LIBRARIES libCore libRIO libNet libHist libGpad libTree libRint libMatrix libMathCore)
   else()
     include(${ROOT_USE_FILE})
+    remove_definitions(${ROOT_DEFINITIONS})
```

Comment:
Consider adding a comment to explain why this is needed (and whether there is a risk of removing a user customization).

Reply:
OK will do (it somehow duplicates some defines, took me quite some time to figure out where it was coming from...)

Reply:
OK, I added a comment. As it tells, this might most probably be fixed on the ROOT side. To be investigated. (and then I'll remove this statement)